pt-BR/mail/chrome/messenger/importMsgs.properties

```diff
@@ -280,12 +280,12 @@
 2135=Observações
 
 # Description: Strings for the import dialog
-ImportMailDialogTitle=Import Mail
-ImportAddressBooksDialogTitle=Import Address Books
-ImportSettingsDialogTitle=Import Settings
-ImportMailListLabel=Select application to import mail from:
-ImportAddressBooksListLabel=Select address book format:
-ImportSettingsListLabel=Select application to import settings from:
+ImportMailDialogTitle=Importar e-mails
+ImportAddressBooksDialogTitle=Importar catálogos de endereços
+ImportSettingsDialogTitle=Importar configurações
+ImportMailListLabel=Selecionar de qual aplicativo deve-se importar os e-mails:
+ImportAddressBooksListLabel=Selecionar o formato do catálogo de endereço:
+ImportSettingsListLabel=Selecionar de qual aplicativo deve-se importar as configurações:
```

Comment:
CORRECTIONS NEEDED it should be ImportSettingsListLabel=Selecionar de qual aplicativo devem as configurações serem importadas